builtin/settingtypes.txt

```diff
@@ -623,10 +623,10 @@ screen_h (Screen height) int 600
 #    Save window size automatically when modified.
 autosave_screensize (Autosave screen size) bool true
 
-#    Fullscreen mode.
+#    Fullscreen mode. (Ignored on Linux)
```

Comment:
`(Does not work on Linux)` to prevent from questions like "why is it ignored?!".

Reply:
Good idea